circle.yml

```diff
@@ -117,7 +117,7 @@ workflows:
       - linux-vapor
       - linux-jwt
       - linux-mysql
-      - linux-postgresql
+      # - linux-postgresql
```

Comment:
Not sure what this does; seems unrelated to this commit?

Reply:
Commenting out that CI job disables the broken test job - there seems to be a CI-level misconfiguration there, so the "red X" Github shows about this PR is inaccurate. I can leave it on  if y'all prefer.

Reply:
Removed that commit!